### PR TITLE
Changes SMES to The Super Magnetic in UI

### LINKED ISSUE #6467: Superconducting-Magnetic-Energy-Storage Power Storage Unit.
Somebody discovered PIN number syndrome.

(it says this on the UI of the SMES: "SMES Power Storage Unit")

### PR DESCRIPTION
**Pull request by Probe1**
Closes https://github.com/d3athrow/vgstation13/issues/6467

### CODE REVIEW

File: code/modules/research/designs/boards/machine_engie.dm [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 /datum/design/smes
 	name = "Circuit Design (SMES) "
-	desc = "Allows for the construction of circuit boards used to build SMES Power Storage Units."
+	desc = "Allows for the construction of circuit boards used to build the Superconducting Magnetic Energy Storage Units."
```
Comment (Kurfursten) [inline code/modules/research/designs/boards/machine_engie.dm]: Adding "The" doesn't seem to make much sense if you're going to make it plural ('units')

Comment (Probe1) [reply code/modules/research/designs/boards/machine_engie.dm]: this is gonna be one of those 1 line changed 4 commits PRs.

Comment (PJB3005) [reply code/modules/research/designs/boards/machine_engie.dm]: Wrong.

**Comment by Probe1** [comment]
We're at 3.  CAN WE MAKE IT 4 COMMITS TO CHANGE A LINE OF TEXT?

**Comment by SysadminJeroen** [comment]
Is that a challenge?

**Comment by Probe1** [comment]
No I will literally pay someone to beat you in real life if you raise a single issue about this PR.

**Comment by SysadminJeroen** [comment]
Please apply these patches.

````
From 45c74a40d6c0ce8d563b652a550df7525bb81645 Mon Sep 17 00:00:00 2001
From: Jeroen de Neef <jeroen52@gmail.com>
Date: Thu, 22 Dec 2016 12:57:27 +0100
Subject: [PATCH 1/2] Revert "I'm not editing anything again"

This reverts commit 58567b091a45cc38fcf2657fa5a8f0c214bdba1b.
---
 code/modules/research/designs/boards/machine_engie.dm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/code/modules/research/designs/boards/machine_engie.dm b/code/modules/research/designs/boards/machine_engie.dm
index 24ca268..84bf3c1 100644
--- a/code/modules/research/designs/boards/machine_engie.dm
+++ b/code/modules/research/designs/boards/machine_engie.dm
@@ -6,7 +6,7 @@
 
 /datum/design/smes
 	name = "Circuit Design (SMES) "
-	desc = "Allows for the construction of circuit boards used to build Superconducting Magnetic Energy Storage Units."
+	desc = "Allows for the construction of circuit boards used to build the Superconducting Magnetic Energy Storage Units."
 	id="smes"
 	req_tech = list(Tc_POWERSTORAGE = 4, Tc_ENGINEERING = 4, Tc_PROGRAMMING = 4)
 	build_type = IMPRINTER
-- 
2.10.0.windows.1

````

````
From 5b4cb04e06d9fddff1cc0c8fd6c140f423ce0354 Mon Sep 17 00:00:00 2001
From: Jeroen de Neef <jeroen52@gmail.com>
Date: Thu, 22 Dec 2016 12:57:36 +0100
Subject: [PATCH 2/2] Revert "Revert "I'm not editing anything again""

This reverts commit 45c74a40d6c0ce8d563b652a550df7525bb81645.
---
 code/modules/research/designs/boards/machine_engie.dm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/code/modules/research/designs/boards/machine_engie.dm b/code/modules/research/designs/boards/machine_engie.dm
index 84bf3c1..24ca268 100644
--- a/code/modules/research/designs/boards/machine_engie.dm
+++ b/code/modules/research/designs/boards/machine_engie.dm
@@ -6,7 +6,7 @@
 
 /datum/design/smes
 	name = "Circuit Design (SMES) "
-	desc = "Allows for the construction of circuit boards used to build the Superconducting Magnetic Energy Storage Units."
+	desc = "Allows for the construction of circuit boards used to build Superconducting Magnetic Energy Storage Units."
 	id="smes"
 	req_tech = list(Tc_POWERSTORAGE = 4, Tc_ENGINEERING = 4, Tc_PROGRAMMING = 4)
 	build_type = IMPRINTER
-- 
2.10.0.windows.1

````

**Comment by Kurfursten** [comment]
I mean, if you want a serious answer, it's still a little bit weird (but more acceptable) that you used "the" in the UI title text.

**Comment by Probe1** [comment]
🔫

**Comment by Intigracy** [comment]
I agree.

Remove.

🔫

**Comment by 9600bauds** [comment]
SMES sounds so much better who cares what it stands for

**Pull request merged**